### PR TITLE
update wad version to download RC2

### PR DESCRIPTION
**Pull request by brendanconnolly**
just updating the version to pull RC2 since it is now available

### CODE REVIEW

**Comment by brendanconnolly** [comment]
CLA signed

File: lib/installer.js [hunk 1]
```diff
@@ -4,49 +4,49 @@ import path from 'path';
...
     throw new Error(`Checksum validation error: expected ${WAD_DL_MD5} but ` +
-                    `got ${downloadedMd5}`);
+      `got ${downloadedMd5}`);
   }
 
   return tempFile;
 }
 
-async function installWAD (msiPath) {
+async function installWAD(msiPath) {
```
Comment (mykola-mokhnach) [inline lib/installer.js]: the space after function name is mandatory. Please run `gulp eslint` locally

**Comment by brendanconnolly** [comment]
I'm going to close and resubmit I forgot the md5 and guid

**Comment by jlipps** [comment]
thanks @brendanconnolly! looking forward to it

**Pull request closed without merging**